### ISSUE TITLE
Error Code

### ISSUE THREAD

**Issue by NirHeaven** [consolexinhun/VarGFaceNet#2]
刚才仔细对照原文看了下，没有一个block的结构是对的。。。最基本的常识性问题都犯错，比如SE模块的激活函数用了ReLU

**Comment by consolexinhun** [comment]
> 刚才仔细对照原文看了下，没有一个block的结构是对的。。。最基本的常识性问题都犯错，比如SE模块的激活函数用了ReLU

你好，网络结构的实现参考博文 [人脸识别论文:VarGFaceNet:VarGFaceNet: An Efficient Variable Group Convolutional Neural Network及其PyTorch实现](https://blog.csdn.net/shanglianlm/article/details/104586878)，实现过程中确实有明显错误，比如 SB Block 的第二个激活函数必须是 Sigmoid，然而因时间关系没有与原论文一一对比，这套代码主要是提供一套 pipline，只需要替换网络结构即可，如果你有 VarGFaceNet 的 pytorch 实现方式，欢迎 PR！
